README.md

```diff
@@ -159,39 +160,48 @@ In the case of an anonymous [optional parameter](#optional-parameters) the type
 toggle(String, ?: Boolean) => Boolean
 ```
 
-### Constructors
-
-Constructors in JavaScript require the `new` keyword. You can identify a constructor signature using the `new` keyword as if you were demonstrating usage:
+### Builtin Types
 
 ```js
-new User({ username: String }) => UserInstance
+Any, Array, Boolean, Function, Number, Object, RegExp, String, Symbol, Void
```

Comment:
No they don't belong here.
I was thinking of having a section just for subs.

Ill commit something along these lines. Gimme a sec.